mailstore-connector/src/com/axonivy/connector/mailstore/MailStoreService.java

```diff
@@ -668,8 +671,11 @@ public static Store openStore(String storeName) throws MessagingException {
 		return store;
 	}
 
-	private static Session getSession(String storeName) {
+	private static Session getSession(String storeName) throws Exception {
 		Properties properties = getProperties(storeName);
+		if (SSLContextConfigure.get().isStartTLSEnabled(properties)) {
+			SSLContextConfigure.get().addIvyTrustStoreToCurrentContext();
+		}

```

Comment:
why do setup a global SSL factory instead of only setting the context for the Mail session? This will corrupt the SSL context of ivy...